RoundsWithFriends/GameModes/DeathmatchHandler.cs

```diff
@@ -71,28 +57,24 @@ public override void OnPlayerLeftRoom(Photon.Realtime.Player otherPlayer)
             {
                 if (!newTeamPoints.Keys.Contains(newTeamIDs[player]))
                 {
-                    newTeamPoints[newTeamIDs[player]] = GM_Deathmatch.instance.teamPoints[player.teamID];
+                    newTeamPoints[newTeamIDs[player]] = GM_TeamDeathmatch.instance.teamPoints[oldTeamIDs[player]];
```

Comment:
Copy pasted the wrong gamemode here I think?

Reply:
good catch again - though this does become irrelevant in the game mode abstraction PR. but I'll fix it here too